firmware/thruna/BMS/Src/App/App_Accumulator.c

```diff
@@ -18,6 +18,7 @@
 #define BALANCE_DEFAULT_DUTY (50) // %
 #define BALANCE_TICKS_ON (100U)
 #define BALANCE_TICKS_OFF (100U)
+#define INTERNAL_CELL_RESISTANCE_OHMS (0.0025f)
 
```

Comment:
Can you provide some way of linking to where you got this number? LiIon cell part number + link to datasheet would be great.